content/docs/command-reference/build.md

```diff
@@ -6,11 +6,16 @@ Python package.
 ## Synopsis
 
 ```usage
-usage: mlem build [options] model [subtype]
+Usage: mlem build builder [options] model
 
-arguments:
-MODEL      Path to model  [required]
-[SUBTYPE]  Type of build. Choices: ['whl', 'pip', 'docker_dir', 'docker']
+Builtin builders:
+- docker
+- docker_dir
+- pip
+- whl
+
+Arguments:
```

Comment:
same here, naming/style - let's keep it unified and consistent across docs

Reply:
fixed